lib/grape-swagger/doc_methods.rb

```diff
@@ -491,7 +497,7 @@ def setup(options)
         end
 
         ops = visible_ops.group_by do |route|
-          @@documentation_class.parse_path(route.route_path, api_version)
+          @@documentation_class.parse_path(route.route_path, api_version, hide_module_from_path)
```

Comment:
Why is this passed in and not assigned as a global `@@hide_module_from_path` like everywhere else?